Jenkinsfile

```diff
@@ -4,7 +4,7 @@ pipeline {
     registryCredential = "dockerhub-inriachile"
     dockerImageName = "lsstts/love-commander:"
     dockerImage = ""
-    LSSTTS_DEV_VERSION = "c0016.001"
+    LSSTTS_DEV_VERSION = "c0018.000"
```

Comment:
In this line, It should be `dev-cycle` variable?

Reply:
Thanks for the review @areyesd14 ! Well, this is part of the things that we haven't extended yet. The Jenkinsfile can't read the `.env` file with the respective environment variables, opposed to the docker-compose that implicit loads the environment variables of the `.env` file. So **no**, it has to be hardcoded until we find a method to extend this.

Reply:
Ahh, that makes sense.